File: c/PRAK305-2310817310004-MSAMILRENDYNORSALEH.c
#include <stdio.h>

int main() {
    int detik;
    
    printf("");
    scanf("%d", &detik);

    int hari = detik / 86400; 
    detik = detik % 86400;
    int jam = detik / 3600;
    detik = detik % 3600;
    int menit = detik / 60;
    detik = detik % 60;

    if (hari > 0) {
        printf("%d hari %02d:%02d:%02d\n", hari, jam, menit, detik);
    } else {
        printf("%02d:%02d:%02d\n", jam, menit, detik);
    }

    return 0;
}

File: python/PRAK305-2310817310004-MSAMILRENDYNORSALEH.py
detik = int(input(""))

hari = detik // 86400
detik = detik % 86400
jam = detik // 3600  
detik = detik % 3600
menit = detik // 60  
detik = detik % 60

if hari > 0:
    print(f"{hari} hari {jam:02d}:{menit:02d}:{detik:02d}")
else:
    print(f"{jam:02d}:{menit:02d}:{detik:02d}")
